packages/firebase_app_check/firebase_app_check_platform_interface/lib/src/apple_provider.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// An enum representing the different types of Apple App Attest providers.
+enum AppleProvider {
+  // The debug provider
+  debug,
```

Comment:
Also should use /// for documentation
```suggestion
/// An enum representing the different types of Apple App Attest providers.
enum AppleProvider {
  /// The debug provider
  debug,
```